File: CollectEats/CollectEats/Models/Restaurant.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace CollectEats.Models;

public partial class Restaurant
{
    public int Id { get; set; }

    public string? Nom { get; set; }

    public string? Telephone { get; set; }

    public int? Utilisateurid { get; set; }

    public int? Adresseid { get; set; }

    public virtual Adress? Adresse { get; set; }

    public virtual ICollection<Horairesouverture> Horairesouvertures { get; set; } = new List<Horairesouverture>();

    [JsonIgnore]
    public virtual ICollection<Plat> Plats { get; set; } = new List<Plat>();

    public virtual Utilisateur? Utilisateur { get; set; }
}

File: PoCCollectEats/src/app/api/models/restaurant.ts
/* tslint:disable */
/* eslint-disable */
import { Adress } from './adress';
import { Horairesouverture } from './horairesouverture';
import { Plat } from './plat';
import { Utilisateur } from './utilisateur';
export interface Restaurant {
  adresse?: Adress;
  adresseid?: null | number;
  horairesouvertures?: null | Array<Horairesouverture>;
  id?: number;
  nom?: null | string;
  plats?: null | Array<Plat>;
  telephone?: null | string;
  utilisateur?: Utilisateur;
  utilisateurid?: null | number;
}
